docs/src/main/asciidoc/datasource.adoc

```diff
@@ -450,19 +466,17 @@ Some databases like H2 and Derby are commonly used in the _embedded mode_ as a f
 The recommended approach is to use the real database you intend to use in production, especially when xref:databases-dev-services.adoc[Dev Services provide a zero-config database for testing], and running tests against a container is relatively quick and produces expected results on an actual environment.
 However, it is also possible to use JVM-powered databases for scenarios when the ability to run simple integration tests is required.
 
-NOTE: While configuring Derby to use the embedded engine works as usual in JVM mode, such an application will not compile into a native executable. It is because Quarkus Derby extensions do not support embedding the entire database engine into a native executable, and it only covers making the JDBC client code compatible with the native compilation step.
 
 ==== Support and limitations
 
 Embedded databases (H2 and Derby) work in JVM mode.
 For native mode, the following limitations apply:
 
-* Derby cannot be embedded into the application in native mode, and only remote connection is supported.
-[IMPORTANT]
-====
-Embedding H2 within your native image is not recommended. 
+* Derby cannot be embedded into the application in native mode; it only supports a remote connection.
+The Quarkus Derby extension allows native compilation of the Derby JDBC client, supporting only remote connections.
```

Comment:
```suggestion
* Derby cannot be embedded into the application in native mode; only a remote connection is supported.
This is because the Quarkus Derby extension only allows native compilation of the Derby JDBC client.
```

Reply:
I would say it like this :-)